Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -259,18 +261,36 @@ def github_status_for_queue(self, change, queue):
                     url = 'https://{}/#/builders/{}'.format(config.BUILDBOT_SERVER_HOST, queue_full_name)
             else:
                 status = GitHubEWS.ICON_BUILD_PASS
+                if is_builder_queue and is_tester_queue:
+                    hover_over_text = 'Built successfully and passed tests'
```

Comment:
The new strings in this PR have a mix of periods / no periods. Would be good to standardize on one.